src/utils/label.jl

```diff
@@ -0,0 +1,61 @@
+"""
```

Comment:
**Organization** Here are little helper functions for naming conventions. I put them all together here so it's easier to be consistent.